src/atc/cosmos/cosmos.py

```diff
@@ -111,3 +118,51 @@ def delete_item(
         db = self.client.get_database_client(self.database)
         cntr = db.get_container_client(TableConfigurator().table_name(table_id))
         cntr.delete_item(id, partition_key=pk)
+
+    def delete_container(self, table_id: str):
+        self.delete_container_by_name(TableConfigurator().table_name(table_id))
+
+    def delete_container_by_name(self, table_name: str):
+        db = self.client.get_database_client(self.database)
+        db.delete_container(table_name)
+
+    def recreate_container_by_name(self, table_name: str):
+        """
+        Delete and recreate the container while preserving properties as
+        far as possible.
+        """
+        db = self.client.get_database_client(self.database)
+
+        for container in db.list_containers():
```

Comment:
Isnt there an easier way to ask the Cosmos db about a table than looping trhough all tables?